WalletWasabi/Bases/ByteArraySerializableBase.cs

```diff
@@ -23,7 +23,7 @@ protected ByteArraySerializableBase()
 
 		public string ToHex() => ToHex(false);
 
-		public string ToHex(bool xhhSyntax = false)
+		public string ToHex(bool xhhSyntax)
```

Comment:
Could you reverse this change? This xhhSyntax is not something someone should know about.

Reply:
Oh, I see. Sorry, leave your modifications.

Reply:
Or maybe just delete the `ToHex()` function and leave only the `ToHex(bool xhhSyntax = false)` in it.